demo/demo.md

```diff
@@ -214,7 +243,42 @@ Be sure to use the `customColor` attribute on the auro-icon component to allow c
   ```html
   <auro-button onclick="alert('YOU CLICKED ME!');">Primary</auro-button>
   <auro-button disabled onclick="alert('YOU CLICKED ME!');">Primary</auro-button>
+  <auro-button loading onclick="alert('YOU CLICKED ME!');">Primary</auro-button>
 
   ```
 
 </auro-accordion>
+
+## Do's and don'ts
```

Comment:
Please wrap your examples... see https://auro.alaskaair.com/components/auro/popover for an example.

Reply:
I see the issue, you need to add the bundle reference to the demo index.html file.